JeffShepherd/GPSammich
# Feature/responsive views

## Is this a fix or a feature? ⬇️

Feature

## What is the change? ⬇️

Update styles.css to effectively render html content in tablet and mobile views

## How should this be tested? ⬇️

Open index.html in a browser and ensure everything looks good in laptop, tablet, and mobile-sized views.
